### PR TITLE
Pixel template making code update

### PR DESCRIPTION
**Pull request by tvami**
#### PR description:
Move the codes that are used to create the (2D)Template and GenErr objects to central CMSSW (instead of living on individual github accounts) 

#### PR validation:
in CondTools/SiPixel/test

`cmsRun SiPixelTemplateDBObjectUploader_cfg.py Fullname=SiPixelTemplateDBObject_phase1_0T_mc_BoR3_v1_bugfix Map=../data/SiPixelTemplateDBObject_0T_phase1_BoR3_v1/mapping.csv TemplateFilePath=CondTools/SiPixel/data/SiPixelTemplateDBObject_0T_phase1_BoR3_v1`

and

`cmsRun SiPixelGenErrorDBObjectUploader_cfg.py Fullname=SiPixelGenErrorDBObject_phase1_0T_mc_BoR3_v1_bugfix Map=../data/SiPixelTemplateDBObject_0T_phase1_BoR3_v1/mapping.csv GenErrFilePath=CondTools/SiPixel/data/SiPixelTemplateDBObject_0T_phase1_BoR3_v1`

#### if this PR is a backport please specify the original PR and why you need to backport that PR:
This is not a PR, and there is no PR planned

cc: @mmusich @tsusa

### CODE REVIEW

**Comment by tvami** [comment]
Hi @christopheralanwest , could you please initiate the tests? Thanks!

File: CondTools/SiPixel/data/SiPixelTemplateDBObject_0T_phase1_BoR3_v1/generror_summary_zp0310.out [hunk 1]
```diff
@@ -0,0 +1,1437 @@
+Dot1_150x100_dj0260a@-375V,0.0T@90d,263K,rh=1.10/0.7,1.3/1.3
```
Comment (mmusich) [inline CondTools/SiPixel/data/SiPixelTemplateDBObject_0T_phase1_BoR3_v1/generror_summary_zp0310.out:1]: Hi @tvami, these huge files do not belong to cmssw. Please remove them and squash your commit history in order not to clog the repo.

File: CondTools/SiPixel/test/run_0T_BoR3.sh [hunk 2]
```diff
@@ -0,0 +1,2 @@
+cmsRun SiPixelTemplateDBObjectUploader_cfg.py Fullname=SiPixelTemplateDBObject_phase1_0T_mc_BoR3_v1_bugfix Map=../data/SiPixelTemplateDBObject_0T_phase1_BoR3_v1/mapping.csv TemplateFilePath=CondTools/SiPixel/data/SiPixelTemplateDBObject_0T_phase1_BoR3_v1
```
Comment (mmusich) [inline CondTools/SiPixel/test/run_0T_BoR3.sh:1]: @tvami maybe this can become a unit test?

**Comment by mmusich** [comment]
test parameters:
 - pull_request = https://github.com/cms-data/CalibTracker-SiPixelESProducers/issues/1

**Comment by mmusich** [comment]
@cmsbuild, please test

**Comment by mmusich** [comment]
@silviodonato it seems that the test parameters at https://github.com/cms-sw/cmssw/pull/30563#issuecomment-655034447 were ignored by the bot. Is the syntax incorrect?

File: CondTools/SiPixel/test/run_0T_BoR3.sh (hunk 2 above)
Comment (tvami) [reply CondTools/SiPixel/test/run_0T_BoR3.sh]: https://github.com/cms-sw/cmssw/commit/bd78af91aa10ffba3af0bade8b4959c0a7baada1

**Comment by mmusich** [comment]
@cmsbuild, please test with cms-data/CalibTracker-SiPixelESProducers#1

**Comment by mmusich** [comment]
@smuzaffar, the static analyzer has some complaints about this PR: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-68bfb5/7770/summary.html, but checking the log is not very clear which change proposed here is triggering the warning. Do you have any hint?

**Comment by ggovi** [comment]
+1

**Comment by silviodonato** [comment]
+1

**Pull request merged**